app/src/androidTest/java/com/github/se/cyrcle/ui/map/MapScreenTest.kt

```diff
@@ -285,4 +286,25 @@ class MapScreenTest {
         .assertIsDisplayed()
         .assertHasClickAction()
   }
+
+  @Test
+  fun testPreviewCard() {
```

Comment:
You did the best that you could to test it so well done :+1:

Reply:
You could improve the test by asserting what is displayed on the card since it is linked to parking1 if I am not wrong.

Reply:
About this. The content of the preview card is clearly not the definitive one. I just did this to have an idea of what the Preview Card will look like. The point of this PR is not to design a Preview Card, but rather to ensure their reliability. This is something that can be addressed when we work on its content during the polish sprint.

Reply:
It was just a suggestion i agree with you and i will make sure to bring it up during the polishing.